porch/pkg/git/package_test.go

```diff
@@ -1,126 +0,0 @@
-// Copyright 2022 Google LLC
-//
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-//      http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-package git
-
-import (
-	"context"
-	"path/filepath"
-	"testing"
-
-	v1 "github.com/GoogleContainerTools/kpt/pkg/api/kptfile/v1"
-	"github.com/GoogleContainerTools/kpt/porch/api/porch/v1alpha1"
-	configapi "github.com/GoogleContainerTools/kpt/porch/api/porchconfig/v1alpha1"
-	"github.com/GoogleContainerTools/kpt/porch/pkg/repository"
-	"github.com/go-git/go-git/v5/plumbing"
-	"github.com/google/go-cmp/cmp"
-)
-
-func (g GitSuite) TestUpstreamLock(t *testing.T) {
```

Comment:
I'm not 100% sure how this works, but this test seems to be using this file here https://github.com/GoogleContainerTools/kpt/blob/main/porch/pkg/git/testdata/drafts-repository.tar which is a zipped up .git file. `GetResources()` doesn't appear to work on the mock repo created from it, so we can't read the Kptfiles. 

I'm not sure the best way to create a mock repo from which we can read resources, and given that I don't see any tests for `GetResources`, it seems like there isn't a precedent for this.